scripts/check-changelog.js

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2021 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+const childProcess = require('child_process');
+const fs = require('fs');
+
+const workspace = process.env.GITHUB_WORKSPACE;
+const event = process.env.GITHUB_EVENT;
+
+const body = JSON.parse(event);
+const baseSHA = body.pull_request.base.sha;
+const headSHA = body.pull_request.head.sha;
+
+const gitLogCommand = `git log --format=format:%H ${baseSHA}..${headSHA}`;
```

Comment:
we could add the first line of the commit message in action to the commit id

Reply:
In this command? Why? That's a git command to compare some commits